tests/model/model.cpp

```diff
@@ -784,11 +784,11 @@ TEST(Model, removeComponentInsensitiveToOrder)
 
     // If the order of operations is switched the behaviour is the same:
 
-    // Get a pointer to the second componet in the parsed model
+    // Get a pointer to the second componet in the parsed model.
```

Comment:
`componet` ==> `component` ?

Reply:
Serves me right for adding the missing full-stop ... ;)

Reply:
83ce6a11